gapic/src/main/com/google/gapid/views/CommandOptions.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gapid.views;
+
+import static com.google.gapid.util.Paths.lastCommand;
+
+import com.google.gapid.models.CommandStream;
+import com.google.gapid.models.ProfileExperiments;
+import com.google.gapid.views.CommandTree.Tree;
+import com.google.gapid.widgets.Widgets;
+
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Menu;
+import org.eclipse.swt.widgets.MenuItem;
+
+public class CommandOptions {
+  private final Menu optionsMenu;
+  private final Menu experimentsMenu;
+  private final Tree tree;
+  private final Widgets widgets;
+
+  public CommandOptions(Control parent, Widgets widgets, Tree tree) {
+    this.optionsMenu = new Menu(parent);
+    this.experimentsMenu = new Menu(this.optionsMenu);
+    this.widgets = widgets;
+    this.tree = tree;
+
+    MenuItem editMenuItem = Widgets.createMenuItem(this.optionsMenu , "&Edit", SWT.MOD1 + 'E', e -> {
+      CommandStream.Node node = this.tree.getSelection();
+      if (node != null && node.getData() != null && node.getCommand() != null) {
+        this.widgets .editor.showEditPopup(this.optionsMenu.getShell(), lastCommand(node.getData().getCommands()),
+            node.getCommand(), node.device);
+      }
+    });
+
+    if(!ProfileExperiments.SHOULD_ENABLE_EXPERIMENTS) {
+      this.tree.setPopupMenu(this.optionsMenu, node -> {
+        if(node.getData() == null) {
+          return false;
+        }
+
+        if(node.getCommand() != null && CommandEditor.shouldShowEditPopup(node.getCommand())) {
+          editMenuItem.setEnabled(true);
+          return true;
+        } else {
+          editMenuItem.setEnabled(false);
+          return false;
+        }
+      });
+
+      return;
+    }
+
+    MenuItem experimentMenuItem = Widgets.createSubMenu(this.optionsMenu, "&Experiments", this.experimentsMenu);
+
+    MenuItem disableAF = Widgets.createCheckMenuItem(this.experimentsMenu, "&Disable Anisotropic Filtering", SWT.MOD1 + 'A', e -> {
+      CommandStream.Node node = this.tree.getSelection();
+      if (node != null && node.getData() != null) {
+        this.widgets.experiments.SetDisableAF(((MenuItem)e.widget).getSelection());
+      }
+    });
```

Comment:
Why is this still here, if we also have a dialog for it?

Reply:
To have a quick access. So that user does not have to open the settings popup every time to just enable one experiment. If we think that this is redundant we can remove it. I would suggest to get devrel opinion on this one.

Reply:
You are making it a dialog and then work around the fact that it is a dialog. Maybe it shouldn't be a dialog in the first place. As I mentioned elsewhere in the PR, I'm ok with keeping it a dialog for now, but remove this menu option from here, it makes no sense here.